test/dlc_tests/ec2/pytorch/inference/test_pytorch_inference.py

```diff
@@ -134,3 +134,11 @@ def test_pytorch_inference_telemetry_gpu(pytorch_inference, ec2_connection, gpu_
 @pytest.mark.parametrize("ec2_instance_type", PT_EC2_CPU_INSTANCE_TYPE, indirect=True)
 def test_pytorch_inference_telemetry_cpu(pytorch_inference, ec2_connection, cpu_only, pt15_and_above_only):
     execute_ec2_inference_test(ec2_connection, pytorch_inference, PT_TELEMETRY_CMD)
+
+
+@pytest.mark.usefixtures("sagemaker")
+@pytest.mark.integration("telemetry")
+@pytest.mark.model("N/A")
+@pytest.mark.parametrize("ec2_instance_type", PT_EC2_GRAVITON_INSTANCE_TYPE, indirect=True)
+def test_pytorch_inference_telemetry_graviton_cpu(pytorch_inference, ec2_connection, graviton_only, pt15_and_above_only):
+    execute_ec2_inference_test(ec2_connection, pytorch_inference, PT_TELEMETRY_CMD)
```

Comment:
nit: add an empty next line

Reply:
`pt15_and_above_only` should not be required here as graviton supports start from PT1.10